secret/store.go

```diff
@@ -79,9 +86,8 @@ func newVaultSecretStore() *secretStore {
 	return &secretStore{client: client, logical: logical}
 }
 
-// GenerateSecretID uuid for new secrets
-func GenerateSecretID() string {
-	return uuid.NewV4().String()
+func generateSecretID(request *CreateSecretRequest) string {
+	return base64.StdEncoding.EncodeToString([]byte(request.Name))
```

Comment:
Maybe a good Hash function (Sha-1, Sha-256) is a better choice:)

Reply:
sha-256 should be good enough

Reply:
So true!